contrib/docker/docker-compose.yml

```diff
@@ -15,10 +15,10 @@ services:
     restart: unless-stopped
     # See the readme for a full documentation of the environment settings
     environment:
-      - SYNAPSE_CONFIG_PATH=/etc/homeserver.yaml
+      - SYNAPSE_CONFIG_PATH=/data/homeserver.yaml
     volumes:
       # You may either store all the files in a local folder
-      - ./matrix-config/homeserver.yaml:/etc/homeserver.yaml
+      - ./matrix-config/:/etc/
```

Comment:
replacing the whole of /etc is going to break things (notably TLS).

Reply:
This is not working either as it creates a directory with the name ``homeserver.yaml`` cause the file does not exist when you first run the container.